apps/ledger-live-desktop/package.json

```diff
@@ -179,6 +179,7 @@
     "@types/lodash": "^4.14.182",
     "@types/react": "^17.0.30",
     "@types/react-redux": "^7.1.24",
+    "@types/react-router": "^5.1.20",
```

Comment:
Do we need this new dependency?

Reply:
Yes it's needed for this line https://github.com/LedgerHQ/ledger-live/pull/2538/files#diff-3711a28d4d4ff1cc485612f4565c9ecdc2cf83d608228c9e87d65809ca546d32R35